frontend/test/metabase/scenarios/question/nested.cy.spec.js

```diff
@@ -214,4 +214,75 @@ describe("scenarios > question > nested", () => {
       .closest(".List-item")
       .contains("Auto binned");
   });
+
+  it.skip("should apply metrics including filter to the nested question (metabase#12507)", () => {
+    const METRIC_NAME = "Discount Applied";
+
+    withSampleDataset(({ ORDERS }) => {
+      cy.log("**-- 1. Create a metric with a filter --**");
+
+      cy.request("POST", "/api/metric", {
+        name: METRIC_NAME,
+        description: "Discounted orders.",
+        definition: {
+          aggregation: [["count"]],
+          filter: ["not-null", ["field-id", ORDERS.DISCOUNT]],
+          "source-table": 2,
+        },
+        table_id: 2,
+      }).then(({ body: { id: METRIC_ID } }) => {
+        // "capture" the original query because we will need to re-use it later in a nested question as "source-query"
+        const ORIGINAL_QUERY = {
+          aggregation: ["metric", METRIC_ID],
+          breakout: [["binning-strategy", ["field-id", 15], "default"]],
```

Comment:
I would probably just use `breakout: [["field-id", ORDERS.PRODUCT_ID]],`
```suggestion
          breakout: [["binning-strategy", ["field-id", ORDERS.TOTAL], "default"]],
```

Reply:
I'm not sure if GitHub is messing up, but there shouldn't be any spans of course. I didn't see that when editing.

Reply:
I still don't see any spans. You might be in some dev tool mode :)